src/@next/Button/Button.tsx

```diff
@@ -3,7 +3,7 @@ import { Spinner } from '../Spinner/Spinner';
 import { Typography } from '../Typography';
 import { StyledButton } from './ButtonStyle';
 
-export type ButtonSize = 'default' | 'slim' | 'large';
+export type ButtonSize = 'default' | 'slim' | 'large' | 'verySlim';
```

Comment:
![image](https://github.com/glints-dev/glints-aries/assets/67429929/76f50216-78e7-4729-bae5-d8958d598697)
I found no styles that can give the wanted spacings for the button (between the label and the border), so I had to create this... wdyt? maybe also document this in Figma?

the spacing is: padding 5px horizontal 2px vertical for large screen, 0px for small screen

Reply:
@michac789 I've just checked the design file and there are some issues with the `button group` & `button` settings on Figma. But it looks like the problems exist on storybook as well. The button group component is somehow removing the padding of plain buttons?
![image](https://github.com/glints-dev/glints-aries/assets/12844836/800cdec4-ed8b-4b12-bab4-d579cb7d92ff)
 Can we just use the slim size instead? I'll make time today to fix the issues on Figma to see if we need to make adjustments to Aries.

Reply:
@michac789 I've just checked the design file and there are some issues with the `button group` & `button` settings on Figma.  Actually, we don't need three different sizes for plain buttons... Anyway, instead of creating 'verySlim', can we adjust the spacing of 'slim' as shown in the screenshot? 
![image](https://github.com/glints-dev/glints-aries/assets/12844836/35022c62-ab5d-4825-ad2c-af9ef1388bcc)

Reply:
And then we can use slim buttons with 4px gap (now it's 8px) @michac789

Reply:
@chiahou sure, so now it has 4px padding in all directions for large screen, how about the spacing for the small screen?